### PR TITLE
Adding level1 tests and extra helper functions

### PR DESCRIPTION
**Pull request by hattan**
This PR includes:

* Updated State based tests for level1
* Extra helpers for TerraformState

### CODE REVIEW

File: tests/level0_test.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// +build level1
```
Comment (HadwaAbdelhalem) [inline tests/level0_test.go:1]: should it be // +build level0 ?

File: tests/level1_test.go [hunk 2]
```diff
@@ -1,32 +1,57 @@
...
+	tfState := NewTerraformState(t, "caf_foundations")
+
+	//act
+	global_settings := tfState.GetGlobalSettings()
+
+	//assert
+	assert.Equal(t, tfState.Environment, global_settings["environment"].(string))
+}
+
+func TestFoundationGlobalSettingsEnvironment(t *testing.T) {
```
Comment (HadwaAbdelhalem) [inline tests/level1_test.go:47]: seems like a duplicate of same test on line 35 ?

Comment (HadwaAbdelhalem) [inline tests/level1_test.go:47]: seems like a duplicate of same test on line 35 ?

File: tests/level0_test.go (hunk 1 above)
Comment (hattan) [reply tests/level0_test.go]: Yes, great catch. It's been updated.

File: tests/level1_test.go (hunk 2 above)
Comment (hattan) [reply tests/level1_test.go]: It Is, copy paste error. I removed one of the duplicates.

Comment (hattan) [reply tests/level1_test.go]: It Is, copy paste error. I removed one of the duplicates.

**Review by HadwaAbdelhalem — APPROVED** [review]

**Pull request merged**